src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,14 @@
 Release History
 ===============
 
+**IoT Central**
+
+* Ensure the resourceType (IotApps) is in camel case.
+* Replace appTemplateName field with name for appTemplate API.
+* Add support for industry and locations field in appTemplate API.
+* Remove the need of checking etag when updating the existing iotc app.
+* Add support for origin and properties field in operations API.
+
```

Comment:
Because our current pipeline automatically generate history notes from PR tile and descriptions, so please don't write the history into history note manually, it should be written into the PR description instead.
If you want more details, please refer to the doc: [link](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests)

Reply:
will remove